examples/update-demo/1-run-web-proxy.sh

```diff
@@ -24,6 +24,9 @@ echo
 echo "    http://localhost:8001/static/"
 echo
 
+export KUBE_REPO_ROOT=${KUBE_REPO_ROOT-$(dirname $0)/../..}
+export KUBECTL=${KUBECTL-$KUBE_REPO_ROOT/cluster/kubectl.sh}
```

Comment:
Question for here and set/access of bash vars throughout: we usually use "${VAR}" in case of spaces (e.g. in paths). Have you tested this is there's a space in your path? Not sure time vs value add to audit for this, but maybe worth a TODO if I'm not totally crazy about this.